client/src/elements/item-preview/item-preview.js

```diff
@@ -64,6 +53,14 @@ export class ItemPreview {
 
   async init() {
     try {
+      // set the preview size options
+      const previewSizes = await this.qConfig.get("previewSizes");
+      this.sizeOptions = previewSizes.map(previewSize => {
```

Comment:
Check if `previewSizes` is defined before using it. If you also keep the defaults then this is not a breaking change.

Reply:
Implemented in https://github.com/nzzdev/Q-editor/pull/119/commits/c06fdb3a27c77b02acb0fa8af192f8f8ea9ef691